x-pack/test_serverless/functional/test_suites/security/ftr/one_discover/context_awareness/cell_renderer.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import kbnRison from '@kbn/rison';
+import expect from '@kbn/expect';
+import path from 'path';
+import { FtrProviderContext } from '../../../../../ftr_provider_context';
+import { SECURITY_ES_ARCHIVES_DIR } from '../../../constants';
+
+export default function ({ getService, getPageObjects }: FtrProviderContext) {
+  const PageObjects = getPageObjects(['common', 'timePicker', 'discover', 'svlCommonPage']);
+  const testSubjects = getService('testSubjects');
+  const dataViews = getService('dataViews');
+  const esArchiver = getService('esArchiver');
+
+  describe('security root profile', () => {
+    before(async () => {
+      await PageObjects.svlCommonPage.loginAsAdmin();
```

Comment:
Why is admin needed?  Is it because the discover page is only supposed be accessed by admins in this context?

Reply:
Changed to Viewer here : https://github.com/elastic/kibana/pull/199818/commits/6d819884b4cbeea9370c8759018dd04633ec4fcf